.vib/odoo/goss/goss.yaml

```diff
@@ -0,0 +1,17 @@
+file:
+  /bitnami/odoo/data:
```

Comment:
I think we should check `/bitnami/odoo`, which is the path in which we are mounting the PVC:

https://github.com/bitnami/charts/blob/de983f69068c65b47e51ebdbec874accee0eb924/bitnami/odoo/templates/deployment.yaml#L85-L87